src/pages/LeeJiSoo/Main/Aside/LeeJiSooCard.js

```diff
@@ -0,0 +1,19 @@
+const LeeJiSooCard = props => {
+  return (
+    <section className="profile">
+      <div className="imgDiv">
+        <img alt="room picture with frog" class="profileImg" src={props.src} />
+      </div>
+      <div className="info">
+        <div>
+          <p>{props.userName}</p>
+          <p>{props.time ? `${props.time}분전에 업로드 했습니다` : ''}</p>
```

Comment:
props.time이 존재하지 않는 경우 해당 텍스트 자체를 나오지 않게 하는 거라면,
p 태그 자체를 다음처럼 조건부로 아예 렌더링 시키지 않을 수도 있음
```suggestion
          {props.time && <p>{props.time}분전에 업로드 했습니다</p>
```